src/Hosting.CommandLine/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,69 @@
+using McMaster.Extensions.CommandLineUtils;
+using McMaster.Extensions.Hosting.CommandLine.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System.Threading.Tasks;
+
+namespace McMaster.Extensions.Hosting.CommandLine
```

Comment:
Nit pick: my personal preference is to put extension methods in the type they are extending so intellisense shows them without having to add a new `using`. In this case, `Microsoft.Extensions.Hosting`

Reply:
Weird, didn't even know it was allowed to add to somebody else's namespace.  Seems odd, and kinda breaks the convention of the folder structure mapping to the namespace.  Other than that i have no objection, and you have a preference, so [lets go with it](https://github.com/natemcmaster/CommandLineUtils/pull/167/commits/1f573c2e589ba4ac2dedf535b3000bcccb689e0e).